extensions/adobe/experience/customerJourneyManagement/message-experienceevent.example.1.json

```diff
@@ -0,0 +1,40 @@
+{
+  "xdm:timestamp": "2020-06-15T10:10:00+00:00",
+  "@id": "https://data.adobe.io/experienceid/5ba0475a-5c31-4fce-b573-8ee222a40f45",
+  "xdm:eventType": "message.feedback",
+  "xdm:identityMap": {
+    "ECID": [
+      {
+        "xdm:id": "68519882713298129995549973016107434638",
+        "xdm:primary": true
+      }
+    ]
+  },
+  "xdm:messageExecution": {
+    "@id": "4218b775-bef3-46b2-aee2-7caae052cf94",
+    "xdm:messageID": "aa440b96-ce65-4ad2-b95d-8c3b51f89bc2",
+    "xdm:messageLabel": "Deal for Gold members",
+    "xdm:externalApplicationID": "0cc85961-487d-49e5-9b6c-01f5630756ac",
+    "xdm:externalApplication": "Journey"
+  },
+  "xdm:messageProfile": {
+    "@id": "4218b775-bef3-46b2-aee2-7caae052cf94",
+    "xdm:channel": {
+      "@id": "https://ns.adobe.com/xdm/channels/email"
```

Comment:
do we need this given xdm:channelConext further below

Reply:
In xdm:channelConext we don't have any channelType field to specify for which channel is this EE for.  The consumer of this schema has to rely on presence of object like "xdm:emailChannelContext" or "xdm:pushChannelContext" to identify whether this messages is for Email or Push channel.

Though we can add an explicit channel type field in "xdm:channelConext", but for completeness of messageProfile mixin which can be used in other schemas(if required) also, I added this property here.